n2y/blocks.py

```diff
@@ -570,8 +570,17 @@ def to_pandoc(self):
         return self.children_to_pandoc()
 
 
-class LinkToPageBlock(WarningBlock):
-    pass
+class LinkToPageBlock(Block):
+    # Invoking warning, for backward compatibility with existing 
+    # installations, which will lack the plugin. 
+    # TODO: right? 
+
+    WarningBlock()
```

Comment:
As best I can tell, you're instantiating a new instance of the `WarningBlock` class within the class definition block. The resulting instance is immediately discarded since it doesn't do anything, but even if it were assigned to something, I'm not sure what the intent would be.

@idolcemia could you expand a bit on what you're expecting this to do and why you'd think it would work?